src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = { ...person };
+  let partnerStatus;
+
+  if (isMarried) {
+    partnerStatus =
+      sex === 'm'
+        ? `${partnerName} is my wife`
+        : `${partnerName} is my husband`;
```

Comment:
Only the word wife/husband is different, so set it conditionally instead of repeating all